generate_brick.py

```diff
@@ -407,7 +407,7 @@ def add_definitions():
     for o in filter(lambda x: x != pc, pointclasses):
         G.add((BRICK[pc], OWL.disjointWith, BRICK[o]))
 
-logging.info("Defining Equipment, System and Location subclasses")
+logging.info("Defining Equipment, Controller, System and Location subclasses")
```

Comment:
Let's remove the "Controller" piece from this comment for now given that we aren't adding controllers (yet)